### DISCUSSION TITLE
route based on from and body

### DISCUSSION THREAD

**Discussion by liorme776** [postalserver/postal#1487] [Feature suggestions]
i'm new to the system, looks cool

but i don't fully understand it
can i do a routing based on from + to + body content?

say, mail sent to from x@x.com to x@y.com (which is hosted on the postal) with body of X, will be forwarded to Y
can this be done?

**Comment by willpower232** [comment]
I don't believe that can be done out of the box.

I imagine the ability to pattern match an email body would be very complicated but if you had a simpler set of messages to match, and a bit of ruby knowledge, you could integrate it yourself.
